src/vs/workbench/browser/parts/compositePart.ts

```diff
@@ -436,6 +437,8 @@ export abstract class CompositePart<T extends Composite> extends Part {
 
 			updateStyles: () => {
 				titleLabel.style.color = $this.titleForegroundColor ? $this.getColor($this.titleForegroundColor) || '' : '';
+				const border = $this.titleBorderColor ? $this.getColor($this.titleBorderColor) : undefined;
+				parent.style.borderBottom = border ? `1px solid ${border}` : '';
```

Comment:
Just setting the border bottom will take away 1px from the title height causing everything to not be centered anymore. We might need to solve this the same way we solve it for editor title (tab bar). We use the ::after element in that case.

See here:
https://github.com/a-stewart/vscode/blob/6b8908833b519af7b37fee9772c6af205dc4fa80/src/vs/workbench/browser/parts/editor/editorGroupView.ts#L2141-L2149
https://github.com/a-stewart/vscode/blob/6b8908833b519af7b37fee9772c6af205dc4fa80/src/vs/workbench/browser/parts/editor/media/editorgroupview.css#L121-L131

Reply:
Resolving this conversation - the approach above wouldn't work but have adjusted the code to handle the centering.